### PR TITLE
Updating Radialmenu to allow walking whilst using

### PR DESCRIPTION
**Pull request by Jamie9192**
**Describe Pull request**
Many requests for walking whilst using menu observed on Discord.
Added Toggle and UseWalking config options

Toggle: false = must hold key, true = toggle menu

UseWalking ture = allows player to walk whilst using radialmenu, false = player movement stops

- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) Yes
- Does your code fit the style guidelines? [yes/no] Yes
- Does your PR fit the contribution guidelines? [yes/no] Yes

### CODE REVIEW

**Comment by tom-osborne** [comment]
@Jamie9192 Thanks for the PR. 
Is the intention of the Toggle to allow F1 to open and also F1 to close the menu, or just to open with one press (not hold)?

**Comment by Jamie9192** [comment]
> @Jamie9192 Thanks for the PR. 
> Is the intention of the Toggle to allow F1 to open and also F1 to close the menu, or just to open with one press (not hold)?

The toggle is set for F1 to open and click centr to close or escape to close. 

Maybe toggle isn't a great description, I will work on making it close with keybind on main.js and resubmit.

**Comment by tom-osborne** [comment]
Understood. I think it's fine if that's the case. We could maybe just make the config option something like this to add clarity:
```lua
Config.HoldKeyToOpen = true  -- [True: Hold KeyBind to open / False: Press Keybind once to open]
```
I think we should also have this for clarity too:
```lua
Config.UseWhileWalking = true
```

File: client/main.lua [hunk 1]
```diff
@@ -186,22 +188,46 @@ local function SetupRadialMenu()
...
+end
 
-    if bool then
-        TriggerEvent('qb-radialmenu:client:onRadialmenuOpen')
-        SetupRadialMenu()
+
+local function setRadialState(bool, sendMessage, delay)
+        -- Menuitems have to be added only once
+    if Config.UseWhilstWalking then
+            if bool then
```
Comment (tom-osborne) [inline client/main.lua]: Fix indentation here

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**